src/dynamics/prognostic_variables.jl

```diff
@@ -28,7 +28,7 @@ end
 $(TYPEDSIGNATURES)
 Initialize the clock with the time step `Δt` in the `time_stepping`."""
 function initialize!(clock::Clock,time_stepping::TimeStepper)
-    clock.n_timesteps = ceil(Int,24*clock.n_days/time_stepping.Δt_hrs)
+    clock.n_timesteps = ceil(Int,24*clock.n_days/(value(time_stepping.Δt_sec)/3600))
```

Comment:
This would become `time_stepping.Δt_sec.value` or could even be skipped (see comment in PR)